src/components/Main/Main.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
+import { Route, Switch } from 'react-router';
+
+import './Main.scss';
+import AdminOrders from '../../pages/AdminOrders';
+import AdminProducts from '../../pages/AdminProducts';
+import CheckOut from '../../pages/CheckOut';
+import Home from '../../pages/Home';
+import Orders from '../../pages/Orders';
+import OrderSuccess from '../../pages/OrderSuccess';
+import Products from '../../pages/Products';
+import ShoppingCart from '../../pages/ShoppingCart';
 
 function Main() {
     return (
-        <main>
-            <h2>This is the Main Component</h2>
-        </main>
+        <div className="main">
+            <Switch>
+                <Route exact path="/checkout" component={CheckOut}/>
+                <Route exact path="/admin-products" component={AdminProducts}/>
```

Comment:
2 things. 
1. Please have paths be in pattern of admin-orders
2.In a constants file create an object that hols all your routes then reference them from there for anywhere you use it. Advantage of this is in situations like this where you have to re-name the path you can change it in one place and not have to do it in a bunch of places